README.md

```diff
@@ -1,24 +1,158 @@
-Serverless IAM via Terraform
-============================
+AWS Serverless Module
+=====================
+[![Terraform][tf_img]][tf_site]
+[![Travis Status][trav_img]][trav_site]
 
-Tuned, locked-down IAM roles for the minimum set of privileges to run the `serverless` framework.
+Get your [serverless][] framework application to AWS, the **right way**.
 
+## Contents
 
-## Installation
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
 
-First, [download Terraform](https://www.terraform.io/downloads.html) and [install the binary](https://www.terraform.io/intro/getting-started/install.html). You should be able to run:
 
-```sh
-$ terraform --version
+- [Overview](#overview)
+- [Concepts](#concepts)
+- [Modules](#modules)
+- [Integration](#integration)
+  - [Reference project](#reference-project)
+  - [Module integration](#module-integration)
+  - [AWS IAM group integration](#aws-iam-group-integration)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Overview
+
+Getting a [serverless][] application all the way to production in AWS **correctly** using **securely** can be quite challenging. In particular, things like:
```

Comment:
Nit: should `using` be `and` in this sentence?

Reply:
Yeah, I'm `using` the wrong word 😛